### ISSUE TITLE
Add a direct link for Google Cloud NetApp Volumes resource

### ISSUE THREAD

**Issue by girigani24** [hashicorp/terraform-provider-google#19766]
### Community Note

Hello,

This is regarding  Google Cloud NetApp Volumes (GCNV) resources in Terraform.

Currently, there is no direct link given to GCNV. The user has to type NetApp in the "search" field to find GCNV. If there is a direct link provided to GCNV resources, it would be helpful to add in our internal documentation and this would guide the users straight to the required resource.

Please do the needful.

Thanks
Satish G

### Terraform Version & Provider Version(s)

Terraform vX.X.X
on <architecture>
+ provider registry.terraform.io/hashicorp/google vX.X.X
+ provider registry.terraform.io/hashicorp/google-beta vX.X.X

### Affected Resource(s)

google_Google Cloud NetApp Volumes

### Terraform Configuration

```tf
```

### Expected Behavior

There should be a direct link for Google Cloud NetApp Volumes for a better navigation.

### Actual Behavior

The link is provided to the resources nested under Google Cloud NetApp Volumes.

### Steps to reproduce

1. `terraform apply`

**Comment by ggtisc** [comment]
Hi @girigani24!

It already exists, this is the URL: https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/netapp_volume

**Comment by girigani24** [comment]
Hello,

I am actually looking for a link in the highlighted area in the attached
screenshot.

Thanks
Satish
[image: image.png]

**Comment by rileykarson** [comment]
@girigani24 could you share the screenshot directly through GitHub? That didn't seem to render.

**Comment by girigani24** [comment]
Attached the screenshot for your reference.
![Screenshot 2024-10-10 4 37 08 PM](https://github.com/user-attachments/assets/7d77aff5-3b28-43b5-a806-f8d406728f18)

**Comment by rileykarson** [comment]
Thank you! I understand this to mean that you'd like a summary page for the product within the provider, rather than needing to link to the list of resources individually.

/cc @SarahFrench, I've kicked this back to triage, but this will likely end up being a HashiCorp request rather than anything provider-specific so sharing to short-circuit a bit.

**Comment by SarahFrench** [comment]
Yeah, this would be a feature request to the Registry team and not something we have control over. An intermediate we _could_ make would be a page in the Guides section (that is generated by MMv1) where we list all resources/data sources grouped by their service

**Comment by girigani24** [comment]
@rileykarson , you are right about my requirement. At first, I want to guide the user to the product and then to its resources. Thank you for the support.
@SarahFrench , thank you!!
